src/widgets/JSONForm/testJSON.js

```diff
@@ -254,13 +257,15 @@ export const testTongaSurvey = {
           title: 'First Name',
           description: 'The first name of the respondent',
           type: 'string',
-          maxLength: 50,
+          maxLength: 2,
+          errorMessage: 'Hey! that names way to long, buddy!',
```

Comment:
This is now the error message for this field. It's not specific to the error - which we can do if we want: https://www.npmjs.com/package/ajv-errors#messages-for-keywords

But I think for simplicity, just adding a single message might be OK?

Reply:
single message is good